site/content/extend/plugins/community_process.md

```diff
@@ -33,12 +38,14 @@ Basic code review of an experimental plugin involves a quick review by a [core c
 
 - When you are ready for your plugin to start this process, post an introduction in the [Integrations and Apps channel](https://community.mattermost.com/core/channels/integrations) on the community server. E.g., "Hi, I have a plugin that I would like to get reviewed. It is called ... It is designed to ... Thanks for reviewing it!" Here is an example: https://community.mattermost.com/core/pl/6dci1ssexjrh9rmdzt4pdpb9zy
 
+
 ### CI System Setup
 
 Setting up the CI system for your plugin will allow continuous testing of your master branch and releases on our testing servers. Master branch testing is done on https://ci-extensions.azure.k8s.mattermost.com/ and release testing is done on https://ci-extensions-release.azure.k8s.mattermost.com/
```

Comment:
```suggestion
Setting up the CI system for your plugin will allow continuous testing of your master branch and releases on our testing servers. Master branch testing is done on [https://ci-extensions.azure.k8s.mattermost.com](https://ci-extensions.azure.k8s.mattermost.com) and release testing is done on [https://ci-extensions-release.azure.k8s.mattermost.com](https://ci-extensions-release.azure.k8s.mattermost.com).
```

Reply:
I don't think the linking is _needed_ here. Hugo will do this on it's own. Any reason to force the linking?

Reply:
Nope, if it's done automatically then this change can be disregarded. I didn't realize that Hugo does that. :)